### PR TITLE
Added opensearch configs for tiered setup

### PR DESCRIPTION
**Pull request by jashaik**
Signed-off-by: jan shahid shaik <jashaik@progress.com>

### Description

Opensearch configurations to be added in the tiered setup in chef-server ugprade document.

https://chefio.slack.com/archives/C07JP2K7F/p1655212785851379

### Issues Resolved

[List any existing issues this PR resolves, or any Discourse or
StackOverflow discussions that are relevant]
#3400
### Check List

- [ ] New functionality includes tests
- [ ] All buildkite tests pass
- [ ] Full omnibus build and tests in buildkite pass
- [ ] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/main/CONTRIBUTING.md#developer-certification-of-origin-dco>
- [ ] PR title is a worthy inclusion in the CHANGELOG

### CODE REVIEW

File: docs-chef-io/content/server/upgrades.md [hunk 1]
```diff
@@ -619,21 +619,75 @@ To upgrade to Chef Infra Server on a tiered Chef Infra Server configuration, do
 
 1. Back up the Chef Infra Server data before starting the upgrade process using [knife-ec-backup](https://github.com/chef/knife-ec-backup).
 
-2. Confirm that the Chef Infra Server services are operational:
+2. Chef Infra Server 14.14 supports external OpenSearch for indexing.
+
+#### Steps To Enable External OpenSearch
```
Comment (PrajaktaPurohit) [inline docs-chef-io/content/server/upgrades.md:624]: Did we mean to add the steps for tiered setup?

Comment (jashaik) [reply docs-chef-io/content/server/upgrades.md:624]: @PrajaktaPurohit As discussed, Now the chef server is with internal opensearch. If that's the case then we don't need to mention any steps explicitly for tiered setup.

File: docs-chef-io/content/server/upgrades.md [hunk 2]
```diff
@@ -619,21 +619,75 @@ To upgrade to Chef Infra Server on a tiered Chef Infra Server configuration, do
...
+
+We recommend migrating your data over reindexing and reconfiguring.
+
+**Migrate Data**
+
+Copy or move your Elasticsearch OSS data and logs directories to the newly installed OpenSearch paths. See OpenSearch's [documentation on upgrading to OpenSearch](https://opensearch.org/docs/latest/upgrade-to/upgrade-to/#upgrade-to-opensearch). 
+
+**Reindex and Reconfigure**
+
+Reindex and reconfigure your database after upgrading to Chef Infra Server 14.13. The duration of this operation will vary depending on your server hardware and the number of node objects on your Chef Infra Server.
```
Comment (vinay-satish) [inline docs-chef-io/content/server/upgrades.md:667]: I think we can specify the version to - 14.13.42

File: docs-chef-io/content/server/upgrades.md [hunk 3]
```diff
@@ -619,21 +619,75 @@ To upgrade to Chef Infra Server on a tiered Chef Infra Server configuration, do
...
+opscode_erchef['search_auth_password'] = "OPEN_SEARCH_PWD"
+```
+
+{{< note >}}
+
+The OpenSearch user should have full access to the cluster, including access to all cluster-wide operations and the ability to write to all indices. We recommend that the user has the admin backend role. 
+
+Please refer to OpenSearch's documentation on [predefined roles](https://opensearch.org/docs/latest/security-plugin/access-control/users-roles/#predefined-roles) and [role mapping configuration](https://opensearch.org/docs/latest/security-plugin/configuration/yaml#roles_mappingyml).
+
+This user must be created on the external OpenSearch cluster. The Chef Infra Server executable cannot be used to create this user on external OpenSearch setups.
```
Comment (vinay-satish) [inline docs-chef-io/content/server/upgrades.md:652]: `Chef Infra Server executable` is bit confusing

**Review by vinay-satish — APPROVED** [review]
LGTM!

**Comment by jashaik** [comment]
Closing this as we don't need it anymore

**Pull request closed without merging**